### ISSUE TITLE
Add a few more items to live_response/process/procfs_information.yaml

### ISSUE THREAD

**Issue by tclahr** [tclahr/uac#35]
Update live_response/process/procfs_information.yaml to collect the following artifacts:

- "ls -l /proc/[0-9]*/cwd" is one of my go-to items for detecting suspicious processes-- when the CWD is /tmp/.ICEd-unix/fooTWUX67 you know you have a problem
- "cat /proc/%line%/stack" can sometimes reveal details of process behavior-- e.g., waiting on a socket, etc
- "cat /proc/%line%/status" has lots of extra process detail, including PPID etc

Please refer to discussion #34

**Comment by tclahr** [comment]
Merged into develop branch via PR #39
